src/main/java/com/wb/payload/Weather_.java

```diff
@@ -0,0 +1,77 @@
+package com.wb.payload;
+
+import java.util.HashMap;
+import java.util.Map;
+import com.fasterxml.jackson.annotation.JsonAnyGetter;
+import com.fasterxml.jackson.annotation.JsonAnySetter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({ "id", "main", "description", "icon" })
+public class Weather_ {
```

Comment:
unikaj stosowania _ w nazwach. Czy rozni sie ten obiekt od `Weather`? nazwa powinna mi to mowic